pieforproviders/pieforproviders
# Delete absences when an attendance is created for any given day

## 💅🏼 What issue does this fix?



5th bullet point on the comment on #1717 - removes absences if an attendance is created for the same date

## 🍂 Before You Submit


## 🧳 Steps to test

- create an absence using the API, console or the CSV import
- create an attendance for the same day (for the same child)
- you should only have one attendance for that child for that day, and it should not be an absence
